package.json

```diff
@@ -5,22 +5,23 @@
   "description": "",
   "keywords": [],
   "scripts": {
+    "start": "II_FETCH_ROOT_KEY=1 vite",
+    "host": "II_FETCH_ROOT_KEY=1 vite --host",
```

Comment:
leftover?

Reply:
No, I added it because it's the shorthand to expose the host on the lan. It reflects the update of the HACKING.md fie.